docs/src/atomistic/reference/systems.rst

```diff
@@ -7,3 +7,5 @@ Systems
 .. autoclass:: metatensor.torch.atomistic.NeighborsListOptions
     :members:
     :special-members: __eq__, __ne__
+
+.. autofunction:: metatensor.torch.atomistic.systems_to_torch
```

Comment:
we should edit the docstring of `System` that is currently pointing to rascaline to point to this one instead